juriscraper/pacer/reports.py

```diff
@@ -380,3 +380,14 @@ def is_pdf_sealed(self, pacer_case_id, pacer_doc_id, pacer_magic_num=None):
         )
         sealed = "You do not have permission to view this document."
         return sealed in r.content
+
+    def is_entry_sealed(
+        self, pacer_case_id, pacer_doc_id, pacer_magic_num=None
+    ):
+        """Check if a docket entry is sealed without trying to actually download
+        it.
+        """
+        r, url = self._query_pdf_download(
```

Comment:
This work functions correctly in district courts and bankruptcy. I'm not sure if we will use this method in appellate courts, but in case it's required, `_query_pdf_download` will need a tweak to support querying appellate documents by using `make_docs1_url` instead.

Reply:
Thanks for bringing this to my attention. I'll work on that 👍

Reply:
@albertisfu  After checking the dataset of sealed documents extracted from the developer database, We found no instances of sealed docket entries in appellate court dockets. We only found sealed documents. As a result, we will use the new helper function exclusively to identify potential sealed docket entries within the docket reports from district courts.

Reply:
Got it. Correct, if we are going to use this helper exclusively for district courts, it'll work properly.